docs/sources/developers/plugins/add-support-for-explore-queries.md

```diff
@@ -0,0 +1,187 @@
++++
+title = "Add support for Explore queries"
+type = "docs"
++++
+
+# Add support for Explore queries
+
+This guide explains how to add support for [Explore]({{< relref "../../features/explore.md" >}}) to an existing data source.
+
+This guide assumes that you're already familiar with how to [Build a data source plugin]({{< relref "../../../../../tutorials/build-a-data-source-plugin.md" >}}).
+
+By adding support for Explore to your data source, users can make ad-hoc queries without the new of a dashboard. This is useful when users want to troubleshoot or to learn more about the data.
+
+To support Explore, your plugin needs to define a _query editor_ for Explore. Optionally, your plugin can also define a _start page_ for Explore.
```

Comment:
```suggestion
To support Explore, your plugin needs to do one or both of the following:
* Define a _query editor_ for Explore. 
* Define a _start page_ for Explore.
```

Reply:
What happens if they do both? Or is this an either/or?

Reply:
The query editor is mandatory, the start page is optional. Your suggestion suggests that you could write only a start page, which is possible, but has little meaning without the query editor.